usaspending_api/common/mixins.py

```diff
@@ -254,6 +256,70 @@ class SuperLoggingMixin(LoggingMixin):
 
     events_logger = logging.getLogger("events")
 
+    """Mixin to log requests - customized to disable DB logging, remove this method to re-enable"""
```

Comment:
I'm confused about what we're disabling here. I'd vote to remove whatever code we have for logging to the db if we're not going to do that anymore. It will live on in GitHub if we really need it, but in the meantime would prefer to keep things clean.

Were we logging extra info to the db that we weren't writing to the file-based logs?